developer-docs-site/README.md

```diff
@@ -1,41 +1,81 @@
-# Website
+# Developer Documentation
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+   - [Installation](#installation)   
+      - [Requirements](#requirements)   
+   - [Fork and clone the Aptos repo](#fork-and-clone-the-aptos-repo)   
+   - [Build the docs locally](#build-the-docs-locally)   
 
-### Installation
+
+This Aptos Developer Documenatation is built using [Docusaurus 2](https://docusaurus.io/). Follow the below steps to build the docs locally and contribute.
+
+## Installation
+
+**IMPORTANT**: These installation steps apply to macOS environment.
+
+### Requirements
+
+Before you proceed, make sure you install the following tools.
+
+- Install [Node.js](https://nodejs.org/en/download/) by executing the below command on your Terminal:
 
 ```
-$ yarn
+brew install node
 ```
 
-### Local Development
+- Install the latest [Yarn](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable) by executing the below command on your Terminal:
 
 ```
-$ yarn start
+brew install yarn
 ```
 
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+## Fork and clone the Aptos repo
 
-### Build
+1. Fork the Aptos Core repo by clicking on the **Fork** on the top right of this repo page:
+https://github.com/aptos-labs/aptos-core
 
-```
-$ yarn build
-```
+2. Clone your fork.
 
-This command generates static content into the `build` directory and can be served using any static contents hosting service.
+  ```
+  git clone https://github.com/<YOUR-GITHUB-USERID>/aptos-core
 
-### Deployment
+  ```
 
-Using SSH:
+## Build the docs locally
 
-```
-$ USE_SSH=true yarn deploy
-```
+1. `cd` into the `developer-docs-site` directory in your clone.
 
-Not using SSH:
+  ```
+  cd aptos-core/developer-docs-site
+  ```
+2. Run `yarn`.
 
-```
-$ GIT_USER=<Your GitHub username> yarn deploy
-```
+  ```
+  yarn
+  ```
+
+3. Start the Yarn server locally. This will also open the locally built docs in your default browser.
+
+> **NOTE**: This step will not generate static html files, but will only render the docs dynamically.
+
+  ```
+  yarn start
+  ```
+
+4. Install Yarn dependencies.
+
+  ```
+  yarn install
+  ```
+5. Finally, build with Yarn.
+
+  ```
+  $ yarn build
+  ```
+
```

Comment:
I did a reset of my directory and didn't have to do these two steps. Are they required?

Reply:
Good point. I see two separate things happening here. The `yarn start` is enough to serve the docs and any source changes will reflect in the served docs. But these won't generate the `build` static html folder. The steps 4, 5 and 6 (right now it says 4,, 5 and 5) will do the extra step of creating the `build` static html folder. 

With this said, I am thinking if it makes sense to distinguish `yarn start` (dynamic docs serving) from these other steps for static html `build` folder. I sort of do it via the Note but it's easy to miss. I think we should call out the static html part because Docusaurus is indeed a static html site generator and it'll be useful for devs to tzip this build directory and take it wherever they want to, for example.